css/index.css

```diff
@@ -1,144 +1,150 @@
-/* Lets take our knowledge of CSS selectors, properties, and inheritance to add some style to the Sweet Eats Bakery website.  
+*{
+  color: #3C373B;
+}
 
-Task 1: Fork this repo. Clone to your computer. 
+h1 {
+  font-family: 'Gaegu', cursive;
+  margin: 20px 0;
+  /*=======  Update the font size to 60px and center the text */
+  font-size: 60px;
+  text-align: center;
+}
 
-Task 2: Explore the CSS.  Look at all the unique selectors.  Find things that are new to you and learn what they do.  Find properties you may not be familiar with and google them.
+h2, h3, h4, h5, h6 {
+  font-family: 'Gaegu', cursive;
+  margin-bottom: 0;
+  padding: 0 10px;
+  /*=======  Update the font size to 30px */
+  font-size: 30px;
+}
 
-Task 3: Update the CSS.  You will notice the CSS comments throughout the file below.  Follow the instructions to complete the Sweet Eats Bakery site.  If you follow the instructions, you should see this as a final result:
+/*======= Create a selector for ALL p tags in the site to have the following styles:
 
-img/sweet-eats-home-page.png
+font-size: 16px;
+font-family: roboto;
+padding: 10px;
+line-height: 1.4;
 
+*/
+ p {
+  font-size: 16px;
+  font-family: roboto;
+  padding: 10px;
+  line-height: 1.4;
+  text-align: center;
+}
 
-***DO NOT ATTEMPT STRETCH UNTIL YOU ARE FINISHED WITH EVERYTHING ELSE!***
+/*======= Create a selector for ALL images in the site to have the following styles: 
 
-Stretch Task: Fork your final work and see how much you can change the design without changing the HTML.  Try to transform the site by only using CSS selectors.
+  border-radius: 10px;
 
-Stretch Task: Look at CSS animations and see if you could integrate them on the navigation hovers.
+*/
+.img src{
+  border-radius: 10px;
+}
```

Comment:
When you are selecting html tags, themselves, there is no leading '.' 
So, for this selector, it would be 
img {
  border-radius: 10px;
}

'src' is in image tags but not the tag itself. You could also format an html image tag like this:
<img alt='some alt text' src='thesource.png' />